### ISSUE TITLE
`/api/projects/{id}/import/predictions` should return `prediction_ids` of uploaded prediction

### ISSUE THREAD

**Issue by bobokvsky** [heartexlabs/label-studio#4819]
**Is your feature request related to a problem? Please describe.**
Same as https://github.com/HumanSignal/label-studio/issues/1598, but with predictions bulk update

**Describe the solution you'd like**
API JSON returns not only `{'created': len(predictions_obj)}`, but also field named`prediction_ids` with ids (order is important).

**Comment by AymanTalkhani** [comment]
@bobokvsky Not sure if you need images labeled for segmentation/bounding boxes, but if you do, have you tried using [InstaLabel](https://www.instalabel.ai/)? All you need to do is upload your dataset, select and label a few samples you'd like to finetune the model on, and hit the 'Train annotator' button. You can then head over to the 'Annotate' tab, click 'Instant Annotation' and have the finetuned model label the rest of your dataset for you. There's even a crowdsourcing option to correct AI predictions for verified ground truth labels. Could be useful for a lot of AI automated computer vision labeling use cases if you're on the lookout.
